zipline/data/us_equity_pricing.py

```diff
@@ -365,6 +364,7 @@ def iterator(iterator=iterator, assets=set(assets)):
         full_table.attrs['last_row'] = last_row
         full_table.attrs['calendar_offset'] = calendar_offset
         full_table.attrs['calendar'] = calendar.asi8.tolist()
+        full_table.flush()
```

Comment:
I wanted to ensure that this was flushed to make it easier to reason about the state of the files on disk when debugging.